src/Microsoft.Sbom.Adapters/Adapters/ComponentDetection/DockerReferenceComponentExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.ComponentDetection.Contracts.TypedComponent;
+using Microsoft.Sbom.Contracts;
+using Microsoft.Sbom.Contracts.Enums;
+using System.Collections.Generic;
+
+namespace Microsoft.Sbom.Adapters.ComponentDetection
+{
+    /// <summary>
+    /// Extensions methods for <see cref="DockerReferenceComponent"/>.
+    /// </summary>
+    internal static class DockerReferenceComponentExtensions
+    {
+        /// <summary>
+        /// Converts a <see cref="DockerReferenceComponent"/> to an <see cref="SBOMPackage"/>.
+        /// </summary>
+        public static SBOMPackage? ToSbomPackage(this DockerReferenceComponent dockerReferenceComponent) => new ()
+        {
+            Id = dockerReferenceComponent.Id,
+            PackageUrl = dockerReferenceComponent.PackageUrl?.ToString(),
+            PackageName = dockerReferenceComponent.Digest,
+            Checksum = new[]
+            {
+                new Checksum
+                {
+                    Algorithm = AlgorithmName.SHA256,
```

Comment:
Is it always SHA256?

Reply:
I actually have that question myself.  I put that in there as a placeholder and am trying to find out where this is officially defined.  I can't quite recall how we knew which algorithm(s) to use for the other types that have checksums.

Reply:
I found in the CD source code that SHA1 is used here.